src/app/containers/Metadata/index.jsx

```diff
@@ -33,6 +33,8 @@ const MetadataContainer = ({
   lang,
   description,
   openGraphType,
+  openGraphImage,
+  openGraphImageAltText,
```

Comment:
Technically because we're using the images for both twitter and facebook, should we remove 'openGraph' from the names, and just have image / imageAltText instead?  We can still revert to the default if these values are null.

Reply:
Ok. I'll update it